apps/prairielearn/src/lib/aws.js

```diff
@@ -86,7 +86,7 @@ module.exports.uploadToS3Async = async function (
   } else {
     logger.verbose(`Uploaded buffer to s3://${s3Bucket}/${s3Path}`);
   }
-  return res;
+  return /** @type {import('@aws-sdk/client-s3').CompleteMultipartUploadCommandOutput} */ (res);
```

Comment:
Let's add a `TODO` here:

```suggestion
  // TODO: remove type case once the following are addressed:
  // https://github.com/aws/aws-sdk-js-v3/issues/5513
  // https://github.com/aws/aws-sdk-js-v3/pull/5512
  return /** @type {import('@aws-sdk/client-s3').CompleteMultipartUploadCommandOutput} */ (res);
```